modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/causalitydatanodes/DistributionZoneCausalityDataNodesTest.java

```diff
@@ -1228,22 +1250,33 @@ private WatchListener createLogicalTopologyWatchListenerToCheckDataNodes(
                 } catch (Exception e) {
                     fail();
                 }
-            }).thenRun(latch::countDown).thenApply(ignored -> null);
+            })
+            .thenRun(latch::countDown)
+            .thenApply(ignored -> {
```

Comment:
Can you simply merge this and the previous line?

Reply:
done